dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganizationResource.java

```diff
@@ -824,6 +825,24 @@ static boolean doesOrganizationExistToUser(Long organizationId, Long userId, Org
     }
 
 
+    static boolean isUserAdminOrMaintainer(Organization organization, Long userId) {
+        OrganizationUser organizationUser = getUserOrgRole(organization, userId);
+        if (organizationUser == null) {
+            return false;
+        }
+
+        if (organizationUser.getRole() == OrganizationUser.Role.ADMIN || organizationUser.getRole() == OrganizationUser.Role.MAINTAINER) {
+            return true;
+        }
+        return false;
+    }
+
+    static boolean isUserAdminOrMaintainer(Long organizationId, Long userId, OrganizationDAO organizationDAO) {
+        Organization organization = organizationDAO.findById(organizationId);
```

Comment:
Presumably this could return null (I think you were checking for that before), which is why you need to check for null above.